content/telegraf/v1.20/administration/commands.md

```diff
@@ -0,0 +1,91 @@
+| `--input-filter <filter>`        | Filter inputs to enable. Separator is `:`.                                                                                        |
+| `--input-list`                   | Print available input plugins.                                                                                                    |
+| `--output-filter`                | Filter outputs to enable. Separator is `:`.                                                                                       |
+| `--output-list`                  | Print available output plugins.                                                                                                   |
+| `--pidfile <file>`               | File to write PID to.                                                                                                             |
+| `--pprof-addr <address>`         | pprof address to listen on. Disabled by default.                                                                                  |
+| `--processor-filter <filter>`    | Filter processors to enable. Separator is `:`.                                                                                    |
+| `--quiet`                        | Run in quiet mode.                                                                                                                |
+| `--section-filter <filter>`      | Filter config sections to output (`agent`, `global_tags`, `outputs`, `processors`, `aggregators` and `inputs`). Separator is `:`. |
+| `--sample-config`                | Print full sample configuration.                                                                                                  |
+| `--once`                         | Gather metrics once, write them, and exit.                                                                                        |
+| `--test`                         | Gather metrics once and print them.                                                                                               |
+| `--test-wait`                    | Number of seconds to wait for service inputs to complete in test or once mode.                                                    |
+| `--usage <plugin>`               | Print plugin usage (example: `telegraf --usage mysql`).                                                                           |
+| `--version`                      | Print Telegraf version                                                                                                            |
+
+## Examples
+
+### Generate a telegraf config file
+
+```sh
+telegraf config > telegraf.conf
+```
+
+### Generate config with only CPU input and InfluxDB output plugins defined
+
+```sh
+telegraf --input-filter cpu --output-filter influxdb config
+```
+
+### Run a single Telegraf collection, outputting metrics to stdout
```

Comment:
is "collection" a commonly used term for all configured plugins?

Reply:
@sjwang90

Reply:
Nah, I think collection should be "configuration"